src/content/en/updates/2019/01/shape-detection.md

```diff
@@ -93,12 +93,20 @@ text. The following bullet list contains examples of use cases for all three fea
 | ------------------------------------------ | ---------------------------- |
 | 1. Create explainer                        | [Complete][explainer]        |
 | 2. Create initial draft of specification   | [In Progress][spec]          |
-| 3. Gather feedback & iterate on design     | [In progress](#feedback)     |
-| **4. Origin trial**                        | **[In progress][ot]**        |
+| 3. Gather feedback & iterate on design     | [**In progress**](#feedback) |
+| **4. Origin trial**                        | [Complete](ot)               |
 | 5. Launch                                  | Not started                  |
 
 ## How to use the Shape Detection API {: #use }
 
+Note: We're currently integrating feedback we gathered now that the Origin Trial has ended
```

Comment:
The OT doesn't end until 74, can you change this to say something to the effect of "The origin trial is expected to end in Chrome 74 and the API will be turned off while we integrate developer feedback."

Reply:
@reillyeon Can you please quickly clarify if the API will continue to be available behind a flag, or if it will be completely removed?

Reply:
It will continue to be available behind the #enable-experimental-web-platform-features flag.